doc/faq.rst

```diff
@@ -29,9 +29,11 @@ script when working on multiple projects at the same time.
 Can I use topkg with dune?
 ==========================
 
-Yes, have a look at the topkg-jbuilder_ project for more details.
+Using topkg with dune is not recommended. Instead, have a look at the
```

Comment:
"not recommended" seems a bit strong here. Maybe a more neutral way to present the situation is to title this "How can I manage my releases" and keep the same information - what you could do with with topkg (and topkg-jbuilder), you can do with dune-release.

Reply:
How about we just say that we recommend using dune-release instead? I think that presenting the two options as equal is doing a disservice to the users as we really know which one is better. The maintainer of topkg-jbuilder himself (@samoht) abandoned it in favor of creating dune-release, I don't think there's much controversy here.

Reply:
Yes, we should recommend dune-release, that's a great tool.

I meant that "Can I use topkg with dune" is a bit too specific: people don't come trying to use topkg, they're presumably just looking for a way to manage their releases.

That being said, it's a very minor point and can be clarified later. This can be merged as is :+1:

Reply:
Yeah, but it's the FAQ and people ask about topkg quite often. I Suppose we should split this into 2 questions:

1) How do I distribute my software with dune?
2) How do I use topkg with dune?

Reply:
That's great, thanks.